manifests/0000_90_kube-apiserver-operator_04_servicemonitor-apiserver.yaml

```diff
@@ -110,8 +110,18 @@ spec:
     rules:
     - alert: UsingDeprecatedAPIExtensionsV1Beta1
       annotations:
-        message: A client in the cluster is using deprecated extensions/v1beta1 API that will be removed soon.
+        summary: "A client is using a deprecated {{"{{"}} $labels.group {{"}}"}}/{{"{{"}} $labels.version {{"}}"}} API version"
+        message: |
+          A client in the cluster has used a deprecated {{"{{"}} $labels.group {{"}}"}}/{{"{{"}} $labels.version {{"}}"}} API version of the {{"{{"}} $labels.resource {{"}}"}} resource in the last 24 hours.
+          This API version will be removed soon; please migrate to the current version of this API.
       expr: |
-        apiserver_request_count{group="extensions",version="v1beta1",resource!~"ingresses|",client!~"hyperkube/.*|cluster-policy-controller/.*"}
+        increase(
+          apiserver_request_count{
+            group="extensions",
+            version="v1beta1",
+            resource!~"ingresses|",
+            client!~"hyperkube/.*|cluster-policy-controller/.*|velero-server/.*"
+          }[24h]
+        ) > 0
       labels:
-        severity: warning
+        severity: none
```

Comment:
why severity none? 🤔

Reply:
I remember that severity should've stayed, from the discussion we were to only change the metrics to be rate based.

Reply:
In the OSD context, Critical and Warnings are fed into monitoring for cluster health. This alert isn't particularly actionable and doesn't actually impact the cluster's current health -- it's an "info" type alert that you should do something prior to the next upgrade, but the current cluster health isn't degraded. None in this case, I believe is appropriate.

Reply:
Note that with a severity of none, it still surfaces in the on-cluster UI

Reply:
> alert isn't particularly actionable and doesn't actually impact the cluster's current health

Precisely why this alert shouldn't exist in the first place.

Reply:
The reasoning behind is that we need to yell loudly at people using this api, b/c otherwise after next upgrade this won't work. I can be convinced and I was originally going after an info level, but none such exists atm. That's why the warning, and it's important one to have.

Reply:
But in 4.4 its not firing, otherwise we would have had failed jobs which confirms @tnozicka:
> note that for 4.4 the serving is disabled so we should drop this check entirely there

Do we want our customers to react on this alert? And if yes how?

Reply:
> Do we want our customers to react on this alert? And if yes how?

If this fires in 4.3 it means you have clients accessing endpoints that will be removed in the next release and they should be updating that code.

Reply:
> I can be convinced and I was originally going after an info level, but none such exists atm.

They do exist. OLM uses `info` level alerts and we also (will) have special rules for handling this severity.

> If this fires in 4.3 it means you have clients accessing endpoints that will be removed in the next release and they should be updating that code.

This is not immediately actionable from operator point of view.